backend/utils/airtable/utils.js

```diff
@@ -80,7 +81,7 @@ async function getCurrentRoundNumber() {
 async function getCurrentRound() {
   const nowDateString = new Date().toISOString();
   const roundParameters = await _getFundingRoundsSelectQuery(
-    `AND({Start Date} <= "${nowDateString}",{Voting Starts} > "${nowDateString}", "true")`
+    `AND({Start Date} < "${nowDateString}", "true")`
```

Comment:
Thank you! :+1: 

Explaining to myself as there are no comments. In this case, we should be expecting a few rows and it should not matter because we'll take [0] index. As we create rounds in-order, this should always work.